libc-test/semver/linux-gnu-reicv64gc.txt

```diff
@@ -0,0 +1,6 @@
+PTRACE_GETFPREGS
```

Comment:
Shouldn't the name of this file be `linux-gnu-riscv64gc.txt`?

Reply:
Oh this was my mistake, I should have checked for these low-level errors before pushing. fixed.